src/isa/riscv64/system/intr.c

```diff
@@ -238,4 +238,11 @@ word_t isa_query_intr() {
     }
   }
   return INTR_EMPTY;
-}
\ No newline at end of file
+}
+
+#ifdef CONFIG_USE_XS_ARCH_CSRS
+word_t INTR_TVAL_SV39_SEXT(word_t vaddr) {
+    vaddr = vaddr & (vaddr_t)0x7FFFFFFFFF;
```

Comment:
use 2 spaces instead of 4 spaces for indentation

Reply:
ok